src/java.net.http/share/classes/jdk/internal/net/http/common/DebugLogger.java

```diff
@@ -276,7 +279,7 @@ public void log(Level level, ResourceBundle unused, String msg,
                 && logLevel.getSeverity() <= severity
                 && logger.isLoggable(level)) {
             logger.log(level, unused,
-                    getFormat(new StringBuilder(), msg, null).toString(),
+                    format(new StringBuilder(), msg, null).toString(),
```

Comment:
`getFormat` is no longer used, can you remove it?

Reply:
Oh - that's an oversight indeed. Thanks for spotting it. Done.